examples/4_circuitobserver.jl

```diff
@@ -0,0 +1,56 @@
+using PastaQ
+using ITensors
+using Random
+
+Random.seed!(1234)
+
+N = 4     # Number of qubits
+depth = 10 # Depth of the quantum circuit
+
+# Generate random quantum circuit built out of
+# random 2-qubit unitaries 
+println("Random circuit of depth $depth on $N qubits:")
+circuit = randomcircuit(N, depth; seed = 1234)
```

Comment:
Does this use an `rng` keyword argument now?

Reply:
For simple examples like this, if you don't want to complicate it by making and passing an `rng`, isn't it sufficient to just call `Random.seed!(1234)` right before calling `randomcircuit`?

Reply:
yes, this is a leftover from the previous design